homeassistant/components/homeassistant_sky_connect/__init__.py

```diff
@@ -75,7 +75,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     if not usb.async_is_plugged_in(hass, matcher):
         # The USB dongle is not plugged in
-        raise ConfigEntryNotReady
+        hass.async_create_task(
+            hass.config_entries.async_set_disabled_by(
+                entry.entry_id, ConfigEntryDisabler.INTEGRATION
```

Comment:
I'm not sure if we need a new type, or if it would have been acceptable to use the existing `ConfigEntryDisabler.USER`

Reply:
A new type makes sense to me. It's not the user that explicitly disables the config entry.